### PR TITLE
Pin to version 0.3.1 of Open SDG

### PR DESCRIPTION
**Pull request by brockfanning**
Pin to a certain version of Open SDG so that unexpected changes won't happen.

### CODE REVIEW

**Comment by dawncomer** [comment]
thanks @brockfanning

**Comment by dawncomer** [comment]
@brockfanning I merged the PR, tried to re-run the nightly build from last night, but it failed.  I'll check tomorrow evening to see if PR work, thanks!

**Comment by brockfanning** [comment]
@dawncomer Ok, yeah that's because when you "re-run" a particular build, it uses the same snapshot of code that it used originally. So, if you re-run a build that failed originally, it's probably going to fail again.

But FYI, there's no need to manually re-run a build after every merge - when you click "Merge" that actually kicks off a build. See build #130 [here](https://circleci.com/gh/dawncomer/open-sdg-site-starter), which is the build that got kicked off when you merged the PR.

In a nutshell, there are 2 ways to do a new build:
1. Merge a PR
2. Wait for the nightly build

**Comment by dawncomer** [comment]
got it! thank @brockfanning
